Sources/SwiftGRPCNIO/ClientCalls/BaseClientCall.swift

```diff
@@ -83,6 +83,12 @@ extension BaseClientCall: ClientCall {
     return self.clientChannelHandler.statusPromise.futureResult
   }
 
+  // Workaround for: https://bugs.swift.org/browse/SR-10128
+  // Once resolved this can become a default implementation on `ClientCall`.
```

Comment:
Would you mind adding a TODO to put this back in place?

Reply:
Done!